references.bib

```diff
@@ -181,3 +181,51 @@ @article{Teshima2006
 volume = {16},
 year = {2006}
 }
+
```

Comment:
@article{Cheng2018,
abstract = {Understanding plant evolution and diversity in a phylogenomic context is an enormous challenge due, in part, to limited availability of genome-scale data across phylodiverse species. The 10KP (10,000 Plants) Genome Sequencing Project will sequence and characterize representative genomes from every major clade of embryophytes, green algae, and protists (excluding fungi) within the next 5 years. By implementing and continuously improving leading-edge sequencing technologies and bioinformatics tools, 10KP will catalogue the genome content of plant and protist diversity and make these data freely available as an enduring foundation for future scientific discoveries and applications. 10KP is structured as an international consortium, open to the global community, including botanical gardens, plant research institutes, universities, and private industry. Our immediate goal is to establish a policy framework for this endeavor, the principles of which are outlined here.},
author = {Cheng, Shifeng and Melkonian, Michael and Smith, Stephen A. and Brockington, Samuel and Archibald, John M. and Delaux, Pierre-Marc and Li, Fay-Wei and Melkonian, Barbara and Mavrodiev, Evgeny V. and Sun, Wenjing and Fu, Yuan and Yang, Huanming and Soltis, Douglas E. and Graham, Sean W. and Soltis, Pamela S. and Liu, Xin and Xu, Xun and Wong, Gane Ka-Shu},
doi = {10.1093/gigascience/giy013},
issn = {2047-217X},
journal = {Gigascience},
number = {7},
volume = {3},
year = {2018},
title = {10KP: A phylodiverse genome sequencing plan},
}

@article{Rhie2021,
abstract = {High-quality and complete reference genome assemblies are fundamental for the application of genomics to biology, disease, and biodiversity conservation. However, such assemblies are available for only a few non-microbial species1,2,3,4. To address this issue, the international Genome 10K (G10K) consortium5,6 has worked over a five-year period to evaluate and develop cost-effective methods for assembling highly accurate and nearly complete reference genomes. Here we present lessons learned from generating assemblies for 16 species that represent six major vertebrate lineages. We confirm that long-read sequencing technologies are essential for maximizing genome quality, and that unresolved complex repeats and haplotype heterozygosity are major sources of assembly error when not handled correctly. Our assemblies correct substantial errors, add missing sequence in some of the best historical reference genomes, and reveal biological discoveries. These include the identification of many false gene duplications, increases in gene sizes, chromosome rearrangements that are specific to lineages, a repeated independent chromosome breakpoint in bat genomes, and a canonical GC-rich pattern in protein-coding genes and their regulatory regions. Adopting these lessons, we have embarked on the Vertebrate Genomes Project (VGP), an international effort to generate high-quality, complete reference genomes for all of the roughly 70,000 extant vertebrate species and to help to enable a new era of discovery across the life sciences.},
author = {Rhie, Arang and McCarthy, Shane A. and Fedrigo, Olivier and Damas, Joana and Formenti, Giulio and Koren, Sergey and Uliano-Silva, Marcela and Chow, William and Fungtammasan, Arkarachai and Kim, Juwan and Lee, Chul and Ko, Byung June and Chaisson, Mark and Gedman, Gregory L. and Cantin, Lindsey J. and Thibaud-Nissen, Francoise and Haggerty, Leanne and Bista, Iliana and Smith, Michelle and Haase, Bettina and Mountcastle, Jacquelyn and Winkler, Sylke and Paez, Sadye and Howard, Jason and Vernes, Sonja C. and Lama, Tanya M. and Grutzner, Frank and Warren, Wesley C. and Balakrishnan, Christopher N. and Burt, Dave and George, Julia M. and Biegler, Matthew T. and Iorns, David and Digby, Andrew and Eason, Daryl and Robertson, Bruce and Edwards, Taylor and Wilkinson, Mark and Turner, George and Meyer, Axel and Kautt, Andreas F. and Franchini, Paolo and Detrich III, H. William and Svardal, Hannes and Wagner, Maximilian and Naylor, Gavin J. P. and Pippel, Martin and Malinsky, Milan and Mooney, Mark and Simbirsky, Maria and Hannigan, Brett T. and Pesout, Trevor and Houck, Marlys and Misuraca, Ann and Kingan, Sarah B. and Hall, Richard and Kronenberg, Zev and Sović, Ivan and Dunn, Christopher and Ning, Zemin and Hastie, Alex and Lee, Joyce and Selvaraj, Siddarth and Green, Richard E. and Putnam, Nicholas H. and Gut, Ivo and Ghurye, Jay and Garrison, Erik and Sims, Ying and Collins, Joanna and Pelan, Sarah and Torrance, James and Tracey, Alan and Wood, Jonathan and Dagnew, Robel E. and Guan, Dengfeng and London, Sarah E. and Clayton, David F. and Mello, Claudio V. and Friedrich, Samantha R. and Lovell, Peter V. and Osipova, Ekaterina and Al-Ajli, Farooq O. and Secomandi, Simona and Kim, Heebal and Theofanopoulou, Constantina and Hiller, Michael and Zhou, Yang and Harris, Robert S. and Makova, Kateryna D. and Medvedev, Paul and Hoffman, Jinna and Masterson, Patrick and Clark, Karen and Martin, Fergal and Howe, Kevin and Flicek, Paul Walenz, Brian P. and Kwak, Woori and Clawson, Hiram and Diekhans, Mark and Nassar, Luis and Paten, Benedict and Kraus, Robert H. S. and Crawford, Andrew J. and Gilbert, M. Thomas P. and Zhang, Guojie and Venkatesh, Byrappa and Murphy, Robert W. and Koepfli, Klaus-Peter and Shapiro, Beth and Johnson, Warren E. and Di Palma, Federica and Marques-Bonet, Tomas and Teeling, Emma C. and Warnow, Tandy and Marshall Graves, Jennifer and Ryder, Oliver A. and Haussler, David and O’Brien, Stephen J. and Korlach, Jonas and Lewin, Harris A. and Howe, Kerstin and Myers, Eugene W. and Durbin, Richard and Phillippy,  Adam M. and Jarvis,  Erich D.},
doi =
issn = {1476-4687}
journal = {Nautre},
title = {Towards complete and error-free genome assemblies of all vertebrate species},
year = {2021},
volume = {592},
number = {7856}, 
pages = {737--746},
doi = {10.1038/s41586-021-03451-0}
}